test/config/logger.config.js

```diff
@@ -0,0 +1,16 @@
+const winston = require("winston");
+
+const todaysDate = new Date();
+let newFilename =
+  todaysDate.getFullYear() + "-" + (todaysDate.getMonth() + 1) + "-" + todaysDate.getDate() + "-" + ".log";
+
+const logger = winston.createLogger({
+  level: "debug",
+  transports: [
+    new winston.transports.File({ filename: newFilename, level: "debug" }),
+    new winston.transports.Console({ level: "info" }),
+  ],
+  format: winston.format.combine(winston.format.timestamp(), winston.format.simple())
```

Comment:
As far as I can see your using trailing comma. E.g. in `app/business-objects/data-for-tests.js`. But here I cannot see it :) Please check the whole PR.